mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -3268,7 +3358,9 @@ struct Conv1DGenerator
 
   /// Entry point that transposes into the common form:
   ///   {{n, strideW * w + dilationW * kw, c}, {kw, c}, {n, w, c}}
-  FailureOr<Operation *> generateDilatedConv(bool flatten = false) {
+  FailureOr<Operation *> generateDilatedConv(uint64_t vecChDimSize = 0,
+                                             bool vecChDimScalableFlag = true,
```

Comment:
Should this default to true? I see it's controlled by the vector `inputScalableVecDims` now, but making it the default here is a little odd.

Reply:
Leftover from experiments, thanks for pointing out!